spec/person_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './person'
+require './book'
+
+describe Person do
+  context 'check for correct values passed through person class ' do
+    new_person = Person.new(16, name: 'Hector', parent_permission: true)
+    # new_person2 = Person.new(16, "Obiwan")
```

Comment:
- [ ] Kindly make sure you remove the commented or unused code lines to make your codebase cleaner and for best practices reasons.